src/AppBundle/Committee/CommitteeFeedHandler.php

```diff
@@ -5,24 +5,42 @@
 use AppBundle\Entity\Adherent;
 use AppBundle\Entity\Committee;
 use AppBundle\Entity\CommitteeFeedItem;
+use AppBundle\Mailjet\MailjetService;
+use AppBundle\Mailjet\Message\CommitteeFeedNotificationMessage;
 use Doctrine\Common\Persistence\ObjectManager;
 
 class CommitteeFeedHandler
 {
     private $manager;
+    private $committeeManager;
+    private $mailJet;
 
-    public function __construct(ObjectManager $manager)
+    public function __construct(ObjectManager $manager, CommitteeManager $committeeManager, MailjetService $mailjet)
     {
         $this->manager = $manager;
+        $this->committeeManager = $committeeManager;
+        $this->mailJet = $mailjet;
     }
 
     public function createMessage(Feed\CommitteeMessage $message, Committee $committee, Adherent $adherent): CommitteeFeedItem
     {
         $message = CommitteeFeedItem::createMessage($committee, $adherent, $message->content);
 
+        $this->sendMessageToFollowers($message, $committee);
+
         $this->manager->persist($message);
         $this->manager->flush();
 
         return $message;
     }
+
+    private function sendMessageToFollowers(CommitteeFeedItem $message, Committee $committee)
+    {
+        $followers = $this->committeeManager->getCommitteeFollowers($committee)->getCommitteesNotificationsSubscribers();
+
+        foreach ($followers as $follower) {
+            /** @var Adherent $follower */
+            $this->mailJet->sendMessage(CommitteeFeedNotificationMessage::create($follower, $message));
```

Comment:
You must send only one message with multiple recipients. See how I've done it with the events.